src/App.tsx

```diff
@@ -14,6 +19,23 @@ const App: React.FC = () => {
         <Routes>
           <Route path="/" element={<Login />} />
           <Route path="registration" element={<Registration />} />
+          <Route
+            path="/projects"
+            element={
+              <AdminLayout>
+                <Project />
+              </AdminLayout>
+            }
+          />
+          <Route path="/forgot_password" element={<ResetPassword />} />
```

Comment:
create a page inside that component will exists

Reply:
fixed